app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Rooms Index</h1>
+
+<ul>
+  <% @rooms.each do |room| %>
+    <li>
+      <strong>Suite:</strong> <%= room.suite %><br>
+      <strong>Nightly Rate:</strong> $<%= room.rate %><br>
+      <strong>Hotel:</strong> <%= room.hotel.name %><br>
+      <strong>Number of Guests:</strong> <%= room.guests.count %>
```

Comment:
While it's a simple calculation, having `count` here is still a calculation that deals with querying the database so we'd rather you encapsulate it into a model method. This also allows you to test it in a more isolated environment (model test) which can be very helpful if this were to get more complicated, i.e. if you had to only count distinct guests or there were other conditions on it. I'd consider this a minor MVC infraction